tls/s2n_config.h

```diff
@@ -55,13 +55,14 @@ struct s2n_config {
 
     uint64_t session_state_lifetime_in_nanos;
 
-    uint8_t use_tickets;
+    unsigned use_tickets:1;
```

Comment:
as with the other struct, can we keep all our bit fields together?

this will allow for better struct packing and member alignment

Reply:
For packing and alignment perspective, absolutely yes. I was thinking these fields and variables were defined by different contributors and used in different places, and keep them close would make lookup easier. 

I would update the code as I have no strong objection to it.